src/plugins/equalheight/reports/pre-a11y-1.json

```diff
@@ -0,0 +1,589 @@
+{
+    "@context": "https://wet-boew.github.io/vocab/context/2023/assessment-report-en.json-ld",
+
+    "@type": ["earl:Assertion", "acr:AssessmentReport"],
+
+    "earl:subject": {
+        "@id": "_:subject",
+        "dct:references": "https://wet-boew.github.io/gcweb-compiled-demos/wetboew-demos/equalheight/equalheight-en.html",
+        "@type": [
+            "earl:TestSubject",
+            "schema:WebPage"
+        ],
+        "dct:description": "Accessibility conformance reports of Equal height",
+        "earl:pointer": {
+            "@type": "oa:CssSelector",
+            "@value": "gc-features"
```

Comment:
I don't think you means to select the custom element `<gc-features>` here. Probably the css selector is `.wb-eqht`

Reply:
fixed